src/mds/MDSRank.cc

```diff
@@ -3203,11 +3203,21 @@ bool MDSRank::evict_client(int64_t session_id,
     return false;
   }
 
+  auto& addr = session->info.inst.addr;
+  {
+    CachedStackStringStream _ss;
```

Comment:
In the luminous backport, the fix suggested by @batrick is to use `std::stringstream` instead.

Reply:
@smithfarm all changes have been done in the PR as well, thanks for the help.